ui/src/common/components/AggregationFilters.jsx

```diff
@@ -32,10 +33,16 @@ class AggregationFilters extends Component {
       aggregations &&
       (numberOfResults > 0 || displayWhenNoResults) && (
         <Row className={rowClassName} type="flex" justify="space-between">
-          {aggregations
+          {staticAggregations
             .entrySeq()
-            .filter(([, aggregation]) => aggregation.get('buckets').size > 0)
-            .sort(AggregationFilters.compareAggregationEntries)
+            .concat(
+              aggregations
```

Comment:
this nested calls are not so elegant you can do,

```
aggregations
 .entrySeq... 
 .filter...
 .sort...
 .splice(0, 0, ...staticAggregations.entrySeq())
```

`splice` have `O(N)` complexity, but since we have upto 5 items in this list, we don't care that much I think.

Otherwise I would throw the `order: 0 or -1`, instead of having these nested calls here.

Reply:
I can't use ```splice``` here because we```entrySeq``` gives a sequence and it doesn't have ```splice``` nor ```unshift```. But I still think this is better than having ```0``` or ```-1```. It's just a ```concat```